src/main.css

```diff
@@ -189,7 +192,11 @@ header {
 header h1 {
   display: inline-block;
   padding-right: var(--spacing);
-  color: var(--darkblueish);
+  color: #121212;
+  background-color: #ffe500;
+  padding: 0px 10px 12px 6px;
+  box-sizing: border-box;
+  width: 380px;
```

Comment:
should this align with the left hand column, which is 375px? https://github.com/guardian/editions-card-builder/pull/16/files#diff-6c6331d530703d2e5a5618a39a83caefR215